app/views/markets/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="center">
+  <%= form_for @market, url: { action: @action } do |f| %>
```

Comment:
Nice job making this dynamic for more than just the new action. I'd recommend moving this out into the `_form` partial and then calling this from within the `new` and `edit` views. It is being used for more than just the `new` action so it doesn't necessarily make sense to only be in the `new` view.